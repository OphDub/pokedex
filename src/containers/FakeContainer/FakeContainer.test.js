import React from 'react';
import { shallow } from 'enzyme';
import { FakeContainer, mapStateToProps, mapDispatchToProps } from './index';

describe('FAKE CONTAINER', () => {
  it('should match the snapshot', () => {
    const renderedComponent = shallow (<FakeContainer />);

    expect(renderedComponent).toMatchSnapshot();
  });
});