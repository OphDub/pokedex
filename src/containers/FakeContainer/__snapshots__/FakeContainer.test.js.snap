// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAKE CONTAINER should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <FakeContainer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <button
        onClick={[Function]}
      >
         FAKE 
      </button>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": " FAKE ",
        "onClick": [Function],
      },
      "ref": null,
      "rendered": " FAKE ",
      "type": "button",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <button
          onClick={[Function]}
        >
           FAKE 
        </button>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": " FAKE ",
          "onClick": [Function],
        },
        "ref": null,
        "rendered": " FAKE ",
        "type": "button",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
